.github/ISSUE_TEMPLATE/custom.md

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
Is this custom template a leftover of the toy demo?

Reply:
Nope.  I'm leaving it as a possible option if for some reason, bug or feature doesn't fit the user's case.  There's still the warning against user questions in it.

Reply:
If you don't find anything prefabricated, try this:

```
name: Open a regular issue
about: Don't see your issue here? Then use this general purpose template.
```